internal/adaptors/secondary/http/espn/get-scores.go

```diff
@@ -8,6 +8,7 @@ import (
 	"math"
```

Comment:
This file is horrific and should be merged in this state. It is disgusting too look or think about and it is a crime against humanity